src/sugar3/speech.py

```diff
@@ -193,7 +193,8 @@ class SpeechManager(GObject.GObject):
     __gsignals__ = {
         'play': (GObject.SignalFlags.RUN_FIRST, None, []),
         'pause': (GObject.SignalFlags.RUN_FIRST, None, []),
-        'stop': (GObject.SignalFlags.RUN_FIRST, None, [])
+        'stop': (GObject.SignalFlags.RUN_FIRST, None, []),
+        'mark': (GObject.SignalFlags.RUN_FIRST, None, (int,))
```

Comment:
[Signal argument type usually end with a comma](https://python-gtk-3-tutorial.readthedocs.io/en/latest/objects.html#create-new-signals) because the `__gsignals__` third argument is expected to be a sequence else a `TypeError` is thrown.

Reply:
Thanks.  Let me rephrase that.  While the upstream documentation implies we should use a `tuple` type, almost all our code in Sugar and Toolkit use a `list` type.  You can see this with `git grep SignalFlags`.  I don't mind if it all changes to `tuple` type, as long as it does it in one go, but mixing the styles is unnecessary and unpleasant.

Reply:
I'll make the required changes, thanks.